packages/ppx_wideopen/ppx_wideopen.0.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+authors: [
+  "Ghiles Ziat <ghiles.ziat@lip6.fr>"
+]
+homepage: "https://github.com/ghilesZ/parsley"
+bug-reports: "https://github.com/ghilesZ/parsley/issues"
+dev-repo: "git+https://github.com/ghilesZ/parsley"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "ppx_deriving"
+  "ppx_tools"
```

Comment:
```suggestion
```
I don't see any use of `ppx_tools` in your code base apart from a commented out section in the `Makefile`. Using ppx_tools and ppx_tools_versioned in the same code is rather rare.